parsl/executors/high_throughput/executor.py

```diff
@@ -656,7 +656,7 @@ def _hold_block(self, block_id):
         for manager in managers:
             if manager['block_id'] == block_id:
                 logger.debug("Sending hold to manager: {}".format(manager['manager']))
-                self.hold_worker(manager['manager'])
+                self._hold_manager(manager['manager'])
```

Comment:
Unrelated to the thrust of this PR, but I see it now so observing it: since this is a blocking request, too bad this loop isn't in bulk.  Or that `_hold_manager` is blocking.

Reply:
yeah, that's probably quite interesting especially for large-node-count blocks (eg 1000 nodes). I'll link this comment from #3368 which talks about reworking this protocol.